docs/installation/integrations/react.md

```diff
@@ -338,11 +336,36 @@ class App extends Component {
 export default App;
 ```
 
+### Vite
+
+Vite requires linked packages to be ESM, and unfortunately, the CKEditor build is not ESM yet (but we're working on it). Therefore, you must modify the `vite.config.js` file to integrate a custom build with Vite. The snippet below will allow you to include the custom build in a Vite bundle. Check out the [Vite docs](https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies) to see more details.
+
+```js
+// vite.config.js
+
+export default defineConfig({
+  optimizeDeps: {
+    include: ['@workspace/ckeditor5-custom-build'],
+  },
+  build: {
+    commonjsOptions: {
+      include: [/@workspace\/ckeditor5-custom-build/, /node_modules/],
+    }
+  }
+})
+```
```

Comment:
I think I get it, but the online builder route doesn't require our plugin, right?

Reply:
Yes, bundle generated by the Online Builder project is ready to use and doesn't require CKEditor-specific plugins.